### ISSUE TITLE
How can I get non-minified version of simple-peer?

### ISSUE THREAD

**Issue by stiv-yakovenko** [feross/simple-peer#857]
(For my debugging purposes)?

**Comment by jzombie** [comment]
https://github.com/feross/simple-peer/blob/master/index.js

**Comment by stiv-yakovenko** [comment]
No, this is not what I need. I need unminified version of simplepeer.min.js, so I can replace it. This one is unpreprocessed, it contains require, which browser doesn't understand.

**Comment by jzombie** [comment]
Try this one, change the build command in package.json to this:

`
 "build": "browserify -s SimplePeer -r . > simplepeer.js"
`

The change removes the minify build command as well as renames the output file.
